Documentation/README_BoundsControl.md

```diff
@@ -156,8 +156,8 @@ Bounds control provides the following events. This example uses these events to
 
 <img src="../Documentation/Images/BoundsControl/MRTK_BoundsControl_Events.png" width="450">
 
-## Elastics (Experimental)
-Elastics can be used when manipulating objects via bounds control. Note that the elastics system is still in experimental state. To enable elastics either link an existing elastics manager component or create and link a new elastics manager via the `Add Elastics Manager` button.
+## Elastics - Experimental
```

Comment:
Was this changed to be linkable? It's interesting seeing this delta but still see the parenthesis approach below (and also for the actual elastics page)

Reply:
i changed it back to parenthesis - i thought i had to change it for linking but actually i didn't (wanted to avoid having the manually defined <a link....> anchor definition as this will show up with link highlight in the final docs page - turned out parenthesis are fine with auto anchor generation:) )